### PR TITLE
Added participant list if you are not attending. Added status to show…

### PR DESCRIPTION
**Pull request by Dr-Horv**
…ings

### CODE REVIEW

**Comment by Dr-Horv** [comment]
Solves #92 and #75

File: frontend/src/routes/SingleShowing/index.js [hunk 1]
```diff
@@ -49,10 +50,14 @@ class Showings extends Component {
   };
 
   renderBoughtOrPendingShowing = () => {
-    const { showing } = this.props;
+    const { showing, me } = this.props;
 
     if (showing.ticketsBought) {
-      return <BoughtShowing showing={showing} openSwish={this.openSwish} />;
+      if(showing.participants.includes(me.id)) {
```
Comment (lindskogen) [inline frontend/src/routes/SingleShowing/index.js]: Does this still work with the new api?

Comment (Dr-Horv) [inline frontend/src/routes/SingleShowing/index.js]: Nope, will have to be fixed, depends on which is merged first

File: frontend/src/Showing.js [hunk 2]
```diff
@@ -9,13 +9,31 @@ import PosterBox from "./PosterBox";
 
 const VerticalPaddingContainer = styled.div`padding: 1em 0;`;
 
+const Status = styled.div`
+  position: absolute;
+  right: 1em;
+  top: 1em;
+`
+
+const capitilized = (s) => s.substring(0, 1).toUpperCase() + s.substring(1);
```
Comment (lindskogen) [inline frontend/src/Showing.js]: capitalised*  

I prefer:

```
s[0].toUpperCase() + s.substring(1)
```

**Pull request merged**